staging/2022/benchmarks/benchmarks_cahps.csv

```diff
@@ -0,0 +1,11 @@
+﻿"MIPS Benchmark Results",,,,,,,,,,,,,,,,,
+Measure_Name,Measure_ID,Submission_Method,Measure_Type,Benchmark,Standard_Deviation,Average,Decile 3,Decile 4,Decile 5,Decile 6,Decile 7,Decile 8,Decile 9,Decile 10,Topped Out,Topped Out By Program,High Priority

```

Comment:
`Standard_Deviation,Average` are added here so that the existing benchmarks scripts can be used. These values are not present in the spreadsheet.